### DISCUSSION TITLE
Error Running First Time "You do not have access ..."

### DISCUSSION THREAD

**Discussion by JRaxworthy** [logancyang/obsidian-copilot#321] [Q&A] [answered]
Just running this for the first time, so its probably basic, but here goes:

Installed Copilot and pasted in my API code from ChatGPT to which I subscribe but I get this "You don't have access to this model" message when I type into the prompt.

What am I doing wrong here?

**Comment by logancyang** [comment]
Could you please check the FAQ

**Reply by JRaxworthy** [reply, marked as answer]
I couldn't find the FAQ but I did in the end, thanks. I have never used Github before. I am just an academic using Obsidian as a tool for my research. All I know about coding is what ChatGPT has shown me!

My next issue is that I can't work out how to use the playground. I would assume that if I can get an API key then that means I have access, but as you say, that is not necessarily the case.  

When I head to the playground it says I have no credit but I do have a Plus subscription, and I can't work it out or how to apply for more, so I just think this whole thing is probably beyond my current computer abilities to set it up.

 Thanks very much for making this available and when I am up with more knowledge I will try again.

**Reply by logancyang** [reply]
I heard OpenAI has moved many accounts to prepaid credit instead of paying as you go at the end of the month. You may be one of those accounts so you may have to put in money as balance before you can call the API. It sucks but they need money I guess.

**Reply by logancyang** [reply]
You can sign up at OpenRouter, that option is free up to a usage limit.
